tools/ratbagctl.in

```diff
@@ -1408,7 +1405,6 @@ class RatbagParserRoot(object):
             ns, rest = subparser.parse_known_args(rest, namespace=ns)
             if old_rest == rest:
                 break
-            pr_debug(ns, ns, rest)
```

Comment:
There are a few more pr_debug in RatbagParser.
You could also remove entirely the pr_debug function while you are at it.

Reply:
done and force-pushed